src/vanta_ledger/main.py

```
@@ -0,0 +1,275 @@
+    CORSMiddleware,
+    allow_origins=settings.ALLOWED_ORIGINS,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+# Security
+security = HTTPBearer()
+
+# Initialize document processor
+document_processor = DocumentProcessor()
```

Ruff (0.12.2)

Undefined name `DocumentProcessor`

(F821)